packages/shared/lib/assets.ts

```diff
@@ -1,61 +1,30 @@
+import { BASE_TOKEN } from '@core/network'
 import { Unit } from '@iota/unit-converter'
 import { convertToFiat, currencies, exchangeRates } from 'shared/lib/currency'
-import { formatStakingAirdropReward } from 'shared/lib/participation/staking'
-import { totalAssemblyStakingRewards, totalShimmerStakingRewards } from 'shared/lib/participation/stores'
 import { activeProfile } from 'shared/lib/profile'
-import { Asset, Token } from 'shared/lib/typings/assets'
+import { Asset } from 'shared/lib/typings/assets'
 import { AvailableExchangeRates, CurrencyTypes } from 'shared/lib/typings/currency'
 import { UNIT_MAP } from 'shared/lib/units'
 import { selectedAccount } from 'shared/lib/wallet'
 import { derived } from 'svelte/store'
-import { StakingAirdrop } from 'shared/lib/participation/types'
 
 export const assets = derived(
-    [
-        exchangeRates,
-        currencies,
-        activeProfile,
-        selectedAccount,
-        totalAssemblyStakingRewards,
-        totalShimmerStakingRewards,
-    ],
-    ([
-        $exchangeRates,
-        $currencies,
-        $activeProfile,
-        $selectedAccount,
-        $assemblyStakingRewards,
-        $shimmerStakingRewards,
-    ]) => {
+    [exchangeRates, currencies, activeProfile, selectedAccount],
+    ([$exchangeRates, $currencies, $activeProfile, $selectedAccount]) => {
         if (!$activeProfile || !$selectedAccount) return []
         const profileCurrency = $activeProfile?.settings.currency ?? AvailableExchangeRates.USD
         const assets: Asset[] = [
             {
-                name: Token.IOTA,
-                rawBalance: $selectedAccount.rawIotaBalance,
+                meta: BASE_TOKEN[$activeProfile.networkProtocol],
+                rawBalance: 0,
```

Comment:
How can we assume this is `0` here?

Reply:
No, this is just temporary until we refactor the balance, this will all change soon.